.dockerignore

```diff
@@ -0,0 +1 @@
+/app/.env
```

Comment:
I think you don't need the folder `test`, `.github` and the files `.travis.yml` `LICENSE` `README.md`.
A general good practice is to ignore everything (`**/*`) and add what you need (by adding a `!` before your file/folder (example in fhir-pipe)

Reply:
Updated!

Reply:
:+1: